app/controllers/admin/conference_controller.rb

```diff
@@ -169,29 +173,12 @@ def edit
       end
     end
 
-    def roles
-      @user = User.new
-      @roles = Role::ACTIONABLES + Role::LABELS
-
-      params[:user] ? (@selection = params[:user][:roles].parameterize.underscore) : (@selection = 'organizer')
-      @role_users = get_users(@selection)
-    end
-
-    def add_user
-      @user = User.find_by(email: params[:user][:email])
-      @selection = params[:role]
-      @user.add_role @selection, @conference
-
-      @role_users = get_users(@selection)
-      render 'roles', formats: [:js]
-    end
-
-    def remove_user
-      @selection = params[:role]
-      @user.revoke @selection, @conference
-
-      @role_users = get_users(@selection)
-      render 'roles', formats: [:js]
+    def create_roles
```

Comment:
Sth goes terribly wrong when I put it in after_save.
I will further investigate. Ideas welcome.